SUPPORTED-FORMATS.md

```diff
@@ -38,7 +38,7 @@ the [Analysis Parsers Library](https://github.com/jenkinsci/analysis-model/).
 | 18 | cpd | cpd() | ![CPD](src/main/webapp/icons/dry-24x24.png) ![CPD](src/main/webapp/icons/dry-48x48.png) | [CPD](https://pmd.github.io/latest/pmd_userdocs_cpd.html) | **/cpd.xml |
 | 19 | cppcheck | cppCheck() | - - | CPPCheck |  |
 | 20 | cpplint | cppLint() | - - | CppLint |  |
-| 21 | csslint | cssLint() | - - | CssLint |  |
+| 21 | csslint | cssLint() | - - | [CssLint](http://csslint.net/) |  |
```

Comment:
Can you add these URLs in the classes `MyPy` and `CssLint` and restart the `ToolLister` test?

See comment in line 1:
```
<!--- DO NOT EDIT - Generated by ToolsLister at 2019-07-04T06:20:34.698-->
```
The file was auto-generated and will be overwritten the next time